fern/pages/changelogs/java-spring/2025-03-05.mdx

```diff
@@ -0,0 +1,4 @@
+## 1.6.0
+**`(feat):`** Add support for explicit nulls in serialization
```

Comment:
Discussed offline - 

It'd be _ideal_ if this didn't affect the spring generator at all. On the response end we'll always provide the user with a `Optional` type (not a `Nullable`) type, so the `Nullable` methods are effectively a no-op. 

We see a couple options here:
1. We add a way for the spring generator to tell the model generator that it should _not_ respect nullable types.
2. We add a feature flag to enable this feature, but _only_ expose it on the SDK generator.

We agree that (1) is more ideal because this feature should be enabled by default if the types are marked as `nullable` in OpenAPI and/or the Fern definition, but if it's sufficiently complex to make that happen (2) is a good compromise.